tests/spicy/types/real/coercion-fail.spicy

```diff
@@ -6,7 +6,7 @@ module Foo;
 global untrue: bool = 1.01;
 global falser: bool = 0.0;
 
-global u64max: real = 0xFFFFFFFFFFFFFFFF;
+# global u64max: real = 0xFFFFFFFFFFFFFFFF; # seems that this may work depending on platform
```

Comment:
Let's add a ticket to add validation to reject this (either on the Spicy or on the HILTI level).